<ui version="4.0" >
 <class>Subclassing</class>
 <widget class="QDialog" name="Subclassing" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qt Designer Form Subclassing</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Class name</string>
     </property>
     <property name="buddy" >
      <cstring>leClassName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;File name</string>
     </property>
     <property name="buddy" >
      <cstring>leFileName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="leFileName" >
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>&amp;Location</string>
     </property>
     <property name="buddy" >
      <cstring>lePath</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLineEdit" name="lePath" >
     <property name="readOnly" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Fil&amp;ter</string>
     </property>
     <property name="buddy" >
      <cstring>leFilter</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="leClassName" />
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="leFilter" />
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2" >
    <widget class="QTreeView" name="tvSlots" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="animated" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tvSlots</tabstop>
  <tabstop>leFileName</tabstop>
  <tabstop>lePath</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Subclassing</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>484</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel" >
     <x>527</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
